activerecord/lib/active_record/encryption/config.rb

```diff
@@ -31,6 +32,7 @@ def set_defaults
           self.excluded_from_filter_parameters = []
           self.previous_schemes = []
           self.forced_encoding_for_deterministic_encryption = Encoding::UTF_8
+          self.hash_digest_class = OpenSSL::Digest::SHA256
```

Comment:
Let's set here for SHA1 otherwise apps loading the 5.2 defaults will have SHA256.